lib/main.js

```diff
@@ -45,7 +54,15 @@ async function run() {
     const file = fs.createReadStream(path)
 
     // console.log(octokit.users.getAuthenticated())
-    const { data: { browser_download_url: browser_download_url }} = await octokit.repos.uploadReleaseAsset({url, headers, name, file})
+    const { data: { browser_download_url: browser_download_url }} = await octokit.rest.repos.uploadReleaseAsset({
+      owner,
+      repo,
+      release_id,
+      headers,
+      name,
+      // @ts-expect-error
+      data: file,
```

Comment:
Data expects a string, but because we set headers correctly I _think_ this should work.

Reply:
Do you know how we can do this? We could also

- Revert [this PR](https://github.com/Shopify/upload-to-release/pull/24)
- Pin the action version in [vscode-shopify-internal](https://github.com/Shopify/vscode-shopify-internal) to 2.0.0